src/containers/oracle.js

```diff
@@ -134,6 +135,7 @@ class OraclePage extends React.Component {
               // Finalize oracle if current block has passed arbitrationEndBlock and threshold is not met
               // Since new oracle is guarranteed to spawn if total amount threshold is met we are not checking total BOT amount here
               if (blockCount > oracle.endBlock) {
+                console.log(`!! blockCount ${blockCount} is greater than endBlock ${endBlock}.`);
                 configName = 'FINALIZING';
```

Comment:
console.log here for debugging purpose. Will clear them up later

Reply:
👍